zerocopy-derive/src/repr.rs

```diff
@@ -171,6 +174,33 @@ define_kind_specific_repr!(
     [Align]
 );
 
+impl EnumRepr {
+    fn as_str(&self) -> Option<&'static str> {
+        Some(match self {
+            EnumRepr::C => "C",
+            EnumRepr::U8 => "u8",
+            EnumRepr::U16 => "u16",
+            EnumRepr::U32 => "u32",
+            EnumRepr::U64 => "u64",
+            EnumRepr::Usize => "usize",
+            EnumRepr::I8 => "i8",
+            EnumRepr::I16 => "i16",
+            EnumRepr::I32 => "i32",
+            EnumRepr::I64 => "i64",
+            EnumRepr::Isize => "isize",
+            EnumRepr::Align(_) => return None,
```

Comment:
Maybe put an `unimplemented!()` here so we know if we accidentally try to serialize an align repr via `ToTokens`?

Reply:
Done, and changed the return type to non-optional